pageserver/src/pgdatadir_mapping.rs

```diff
@@ -1921,6 +1937,10 @@ impl<'a> DatadirModification<'a> {
                 return;
             }
         }
+        self.pending_bytes += match &val {
+            Value::Image(inner) => inner.len(),
+            Value::WalRecord(_) => 100, // Rough approximation of typical serialized WalRecord size.
```

Comment:
Curious how you came up with this number

Reply:
From a while back when I was doing some experiments... but it doesn't need to be here.  In e94eb614ebcfcd3759d32391568714d83750eb70 this changes to be the calculated serialized size.